pkgs/desktops/gnome-3/misc/evolution-ews/default.nix

```
@@ -0,0 +1,50 @@
+{ stdenv, fetchgit, gnome3, cmake, gettext, intltool, pkgconfig, evolution-data-server
+, sqlite, gtk3, webkitgtk, libgdata, ... }:
```

Ellipsis not used. `callPackage` will only fill in the available arguments anyway.

```suggestion
, sqlite, gtk3, webkitgtk, libgdata }:
```